packages/@apphosting/adapter-angular/src/bin/build.spec.ts

```diff
@@ -69,6 +76,11 @@ env:
   - variable: SSR_PORT
     value: "8080"
     availability: RUNTIME
+metadata:
+  adapterNpmPackageName: \\@apphosting/adapter-angular
+  adapterVersion: ${packageVersion}
+  framework: angular
+  frameworkVersion: 17.3.2
```

Comment:
I know this test was already here but can we reconfigure this testcase to ONLY test the SSR_PORT variable? As it stands, every time any change is made to the bundle.yaml output, this test needs to change which means it's testing things that it should not be

go/unit-testing-practices#behavior-testing-examples

Reply:
That's a good point, added a `validateFileExistsAndContains` helper and modified the env var test to only test SSR_PORT.